src/Relay/Connection/Connection.php

```diff
@@ -135,4 +129,16 @@ public static function getPageInfoType()
         ]);
     }
 
+    public static function getEdges($value)
+    {
+      return isset($value['edges']) ? $value['edges'] : null;
+    }
+
+    public static function getPageInfo($value) {
+      return isset($value['pageInfo']) ? $value['pageInfo'] : null;
+    }
+
+    public static function getNode($value) {
+      return isset($value['node']) ? $value['node'] : null;
```

Comment:
Indentation should be 4 spaces. Also the opening braces should be on a new line.